### ISSUE TITLE
Other waveforms?

### ISSUE THREAD

**Issue by unfa** [oxesoft/oxefmsynth#18]
I wonder if it'd be possible to allow using other waveforms apart from sine waves?
I think It'd really kick up it's possibilities.

I have no idea if that's feasible with the synth's internal design.

**Comment by oxesoft** [comment]
Yes, it is totally feasible. Are you talking about calculated waveforms or arbitrary ones?

**Comment by unfa** [comment]
I see that Oxe FM synth supports multiple waveforms now! That's awesome!

**Comment by samboy** [comment]
Since this does not appear to be documented anywhere except in `buffers.cpp`, the waveforms are:

* Sine
* Saw
* Triangle
* Pulse
* Band limited pulse
* Band limited saw

More can easily enough be added.
